application/src/main/resources/thingsboard.yml

```diff
@@ -364,7 +374,7 @@ sql:
   # Threshold of slow SQL queries to log. The value set in milliseconds
   log_queries_threshold: "${SQL_LOG_QUERIES_THRESHOLD:5000}"
   # Enable/Disable logging statistic information about tenants
-  log_tenant_stats: "${SQL_LOG_TENANT_STATS:true}"
+  log_tenant_stats: "${SQL_LOG_TENANT_STATS:false}"
```

Comment:
Bunun gibi runtime'da değişmesi gerekenleri not edip runtime'da değiştirelim, default config'i değiştirmeyelim

Reply:
Düzeltildi